streams-tests/jvm/src/test/scala/zio/stream/SinkSpec.scala

```diff
@@ -1109,6 +1127,15 @@ class SinkSpec(implicit ee: org.specs2.concurrent.ExecutionEnv) extends TestRunt
     unsafeRun(sinkIteration(sink, 1).map(_ must_=== ((None, Chunk.empty))))
   }
 
+  private def untilLeftoverAppend = {
```

Comment:
```suggestion
  private def untilOutputLeftoverAppend = {
```

Reply:
Thanks @iravid If there are any other combinators that append leftovers and are not covered here, please let me know